README.md

```diff
@@ -1,12 +1,12 @@
 # vue-example-project
 
-This repo is an example of using [Vue 3](https://vuejs.org/) / [Vite](https://vitejs.dev/) along with [Nord Design System's](https://nordhealth.design/) [components](https://nordhealth.design/components/). Typescript is used, but this is not a necessity for using Vue and Nord together.
+This repo is an example of using [Vue 3](https://vuejs.org/) with [Vite](https://vitejs.dev/) along with [Nord Design System's](https://nordhealth.design/) [components](https://nordhealth.design/components/). TypeScript is used, but this is not a necessity for using Vue and Nord together.
 
-This repo can be forked as a starting point for new apps. However, you may wish to undertake the process yourself so that all dependencies are up to date, and you can choose which Vue features you would like to use. The commit history shows the steps taken to integrate Vue and Nord. Those steps are described next.
+This repo can be forked as a starting point for new apps. However, you may wish to undertake the process yourself so that all dependencies are up to date, and you can choose which Vue features you would like to use. The steps to integrate React and Nord are described next.
```

Comment:
```suggestion
This repo can be forked as a starting point for new apps. However, you may wish to undertake the process yourself so that all dependencies are up to date, and you can choose which Vue features you would like to use. The steps to integrate Vue and Nord are described next.
```

Reply:
Oh oops, thanks!

Reply:
Adjusted